src/fastoad/openmdao/variables.py

```diff
@@ -364,6 +365,11 @@ def update(self, other_var_list: "VariableList", add_variables: bool = True):
 
         for var in other_var_list:
             if add_variables or var.name in self.names():
+                # To avoid to loose variables description when the variable list is updated with a
+                # list without descriptions (issue # 319)
+                if var.name in self.names():
+                    if self[var.name].description and not var.description:
+                        var.description = self[var.name].description
```

Comment:
Is this feature really needed here ?
I do not think that this `update` method should be responsible for filling the holes. If a variable with no description is provided, the expected behavior will intuitively be that it it still won't have any description afterward (and if it does the contrary, the docstring should mention it).

For me, getting missing descriptions should be done when generating the variable list provided as input.
And normally, it is already done when instantiating Variable instances, or when instantiating a VariableList from a Problem, etc...

(but if it happens this change is actually needed, please write `lose` instead of `loose`)

Reply:
I noticed that if I remove those lines, any description provided in the XML input file and now read by your PR #359 are lost if no descriptions are provided within the `desc` kargs of component variables. Is it the behaviour you expected? Because in that case to read the comments in the input file lost its interest. Besides I noticed that keeping those lines active ensure that if a description for a connected variable is provided in the input file but not `desc` field is full the description of the input file is kept but for unconnected variable it is lost. For the moment I've just added a test case in the "tests" directory we can play with to be sure of the final behaviour we want and I corrected the typo. For the moment I do not know which behaviour is better so I let the subject open to discussion ;)